.circleci/config.yml

```diff
@@ -74,6 +76,16 @@ install_docker_buildx: &install_docker_buildx
     # Verify buildx is available
     docker buildx create --use
 
+# Installation script for the docker compose plugin. See: https://docs.docker.com/compose/install/#install-compose-on-linux-systems
+install_docker_compose: &install_docker_compose
+  name: install docker compose
+  command: |
+    mkdir -p ${DOCKER_CONFIG}/cli-plugins
+    curl -SL https://github.com/docker/compose/releases/download/"${DOCKER_COMPOSE_VERSION}"/docker-compose-linux-x86_64 -o ${DOCKER_CONFIG}/cli-plugins/docker-compose
+    chmod +x ${DOCKER_CONFIG}/cli-plugins/docker-compose
+    chmod +x /usr/local/lib/docker/cli-plugins/docker-compose
```

Comment:
File `/usr/local/lib/docker/cli-plugins/docker-compose` may not exists since `curl` download file to `${DOCKER_CONFIG}/cli-plugins/docker-compose`

Reply:
@denis256 I'm not sure what you mean here?

Reply:
I'll try to test this locally/on my own CircleCI setup to make sure it works! Bit of a slow feedback loop on this since I can't trigger the pipeline to test.